deploy/olm-catalog/infinispan-operator.clusterserviceversion.yaml

```diff
@@ -32,119 +43,74 @@ metadata:
             "clusterName": "example-infinispan",
             "name": "mycache"
           }
+        },
+        {
+          "apiVersion": "infinispan.org/v2alpha1",
+          "kind": "Restore",
+          "metadata": {
+            "name": "example-restore"
+          },
+          "spec": {
+            "cluster": "example-infinispan",
+            "container": {
+              "cpu": "1000m",
+              "extraJvmOpts": "-Djava.property=me",
+              "memory": "1Gi"
+            },
+            "path": "asdasd"
+          }
         }
       ]
+    capabilities: Full Lifecycle
+    categories: Database
+    repository: https://github.com/infinispan/infinispan-operator
     description: Create and manage Infinispan clusters.
-    containerImage: quay.io/infinispan/operator:latest
+    containerImage: quay.io/infinispan/operator:2.0.5.Final
```

Comment:
@rigazilla is 2.0.5 correct operator image?

Reply:
Uh no no, let me tag the master branch for a 2.1.0.Alpha. Thanks @dmvolod

Reply:
I'll tag after this merge